wgpu/src/lib.rs

```diff
@@ -1036,14 +1061,15 @@ pub struct Operations<V> {
     /// How data should be read through this attachment.
     pub load: LoadOp<V>,
     /// Whether data will be written to through this attachment.
```

Comment:
I think we also need to change this comment/the other one to mention the fact that the resolve texture is _always_ stored to, no matter what this is set to.

Reply:
good idea

Reply:
This doesnt' seem to have been addressed.